cmd/rsagen/main.go

```diff
@@ -0,0 +1,157 @@
+// The rsagen command generates rsa.PublicKey variables from PEM-encoded
+// RSA public keys.
+package main
+
+import (
+	"bytes"
+	"crypto/rsa"
+	"embed"
+	"flag"
+	"fmt"
+	"go/format"
+	"io"
+	"io/fs"
+	"log"
+	"os"
+	"text/template"
+
+	"github.com/gotd/td/internal/crypto"
+)
+
+//go:embed _template/*.tmpl
+var embedFS embed.FS
+
+var funcs = template.FuncMap{
+	"fingerprint": func(pubkey *rsa.PublicKey) string {
+		return fmt.Sprintf("%08x", uint64(crypto.RSAFingerprint(pubkey)))
+	},
+	"chunks": func(b []byte, size int) [][]byte {
+		var chunks [][]byte
+		for len(b) > size {
+			chunks = append(chunks, b[:size])
+			b = b[size:]
+		}
+		if len(b) != 0 {
+			chunks = append(chunks, b)
+		}
+		return chunks
+	},
+	"single": func(keys []*rsa.PublicKey) (*rsa.PublicKey, error) {
+		if count := len(keys); count != 1 {
+			return nil, fmt.Errorf("expected single key, got %d keys", count)
+		}
+		return keys[0], nil
+	},
+}
+
+func main() {
+	os.Exit(main1())
+}
+
+func main1() int {
+	log.SetFlags(log.Llongfile)
+	var (
+		inPath       = ""
+		outPath      = ""
+		tplPath      = ""
+		tplName      = "main.tmpl"
+		pkgName      = "main"
+		varName      = "PK"
+		singleMode   = false
+		formatOutput = true
+	)
+	flag.StringVar(
```

Comment:
Please use (at least for consistency with other commands):
```
var (
    inPath = flag.String() // ...
)
```
The `StringVar` patter is too verbose.